usr/include/slotmgr.h

```diff
@@ -33,10 +33,17 @@
 #define OCK_HSM_MK_CHANGE_PATH CONFIG_PATH "/HSM_MK_CHANGE"
 #define OCK_HSM_MK_CHANGE_LOCK_FILE LOCKDIR_PATH "/LCK..HSM_MK_CHANGElock"
 
-#define PROC_SOCKET_FILE_PATH "/run/opencryptoki/pkcsslotd.socket"
-#define ADMIN_SOCKET_FILE_PATH "/run/opencryptoki/pkcsslotd.admin.socket"
+#if defined(_AIX)
+#define RUN_ROOT "/var"
```

Comment:
indention, also else case ?

Reply:
`/run` does not exist on AIX, while it does on Linux. if `#else` is omitted then line 42 will error out on platforms other than AIX because `RUN_ROOT` is undefined.

Reply:
I did not mean to omit the else case, I just meant that the indention should also be fixed for the else case.